monosi/monitors/table.py

```diff
@@ -90,7 +90,11 @@ def info(self):
 
     @classmethod
     def validate(cls, monitor_dict):
-        pass
+        if 'columns' not in monitor_dict or len(monitor_dict['columns']) == 0:
+            return False
+
+        # TODO: Better validation
```

Comment:
100% agree that we need to add more validation down the road, just checking the columns is fine for now